sphinx/themes/basic/static/searchtools.js_t

```diff
@@ -256,7 +256,7 @@ var Search = {
             displayNextItem();
           });
         } else if (DOCUMENTATION_OPTIONS.HAS_SOURCE) {
-          $.ajax({url: DOCUMENTATION_OPTIONS.URL_ROOT + '_sources/' + item[0] + '.txt',
+          $.ajax({url: DOCUMENTATION_OPTIONS.URL_ROOT + '_sources/' + item[5] + '.txt',
```

Comment:
Thanks!

I added a commit that only adds `.txt` if it isn't already there: 71e1aaf3770831df1b5b7f36aaadb789176a2927.

What's left to do is to change `'.txt'` to the value of `html_sourcelink_suffix` from `conf.py`.
How do I get this from inside the JavaScript function?